ui/tests/integration/components/auth-form-test.js

```diff
@@ -140,18 +143,11 @@ module('Integration | Component | auth form', function(hooks) {
     });
 
     await render(hbs`{{auth-form cluster=cluster }}`);
-    return settled().then(() => {
-      assert.equal(component.tabs.length, 2, 'renders a tab for userpass and Other');
-      assert.equal(component.tabs.objectAt(0).name, 'foo', 'uses the path in the label');
-      assert.equal(component.tabs.objectAt(1).name, 'Other', 'second tab is the Other tab');
-      assert.equal(
-        replaceSpy.getCall(0).args[0].queryParams.with,
-        'foo/',
-        'calls router replaceWith properly'
-      );
```

Comment:
No more `replaceSpy`?

Reply:
Ran into an error that was making things go boom when we called replaceWith in this component, so the component changed to directly set the param and now the routing happens via two-way binding (something that didn't work when I first wrote it which is why we were doing routing in a didRender hook). Since we're no longer calling the service directly, that assertion wasn't needed. That makes me wonder if there's access to the routing service in the new test setup now...